utils/render-rules.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python3
+
+from glob import glob
+import collections
+import os
+import re
+import pathlib
+
+import argparse
+
+import ssg.build_yaml
+import ssg.controls
+import ssg.yaml
+import ssg.jinja
+import template_renderer
+
+
+class HtmlOutput(template_renderer.Renderer):
+    TEMPLATE_NAME = "rules-template.html"
+
+    def _get_all_compiled_profiles(self):
+        compiled_profiles = glob(str(self.built_content_path / "profiles" / "*.profile"))
+        profiles = []
+        for p in compiled_profiles:
+            profiles.append(ssg.build_yaml.Profile.from_yaml(p))
+        return profiles
+
+    def _set_rule_profiles_membership(self, rule, profiles):
+        rule.in_profiles = []
+        for p in profiles:
+            if rule.id_ in p.selected:
+                rule.in_profiles.append(p)
+
+    def _resolve_var_substitutions(self, rule):
+        rule.description = re.sub('<sub\s+idref="([^"]*)"\s*/>', r"<tt>$\1</tt>", rule.description)
```

Comment:
it requires a comment

Reply:
Do you mean a code comment, or that the generated HTML should feature a tooltip or something like that?

Reply:
A code comment

Reply:
I get it, that's a meaningful requirement.